### DISCUSSION TITLE
Where is the best place to ask questions

### DISCUSSION THREAD

**Discussion by skeets20** [ToolJet/ToolJet#2112] [Q&A] [answered]
Wondering where the best place is to have discussions and ask questions that could also benefit others?  Slack seems pretty quiet.   I'm looking for some guidance on certain functionalities but don't want to pester any one person.

**Comment by withshubh** [comment, marked as answer]
Hey @skeets20 👋 

Currently, we are using [Slack](https://join.slack.com/t/tooljet/shared_invite/zt-r2neyfcw-KD1COL6t2kgVTlTtAV5rtg) as the only communication platform for the **team and the community members**. You can always shoot your questions in the Slack channel and someone from the team will help you out.
If you think that you have a problem that might be helpful later for other community members then I'd suggest you **create a GitHub Discussion** (just like you created this) and we'll help you out.

I hope this helps. ✨
